applications/nrf_desktop/configuration/nrf52840gmouse_nrf52840/child_image/mcuboot/prj_mcuboot_smp.conf

```diff
@@ -36,3 +36,6 @@ CONFIG_CBPRINTF_NANO=y
 CONFIG_TIMESLICING=n
 CONFIG_ARM_MPU=n
 CONFIG_THREAD_STACK_INFO=n
+
+# Use minimal C library instead of the Picolib
+CONFIG_MINIMAL_LIBC=y
```

Comment:
Please make sure that you did not skip any of the child image projects

Reply:
I think I updated them all (I double checked). Still please double check and notify me if I missed anything.